source/features/no-pr-commit-notification.tsx

```diff
@@ -0,0 +1,28 @@
+import * as pageDetect from 'github-url-detection';
+
+import features from '../feature-manager.js';
+import observe from '../helpers/selector-observer.js';
+
+function trimLink(link: HTMLAnchorElement): void {
+	link.pathname = link.pathname.replace(/\/files\/[\da-f]{40}..[\da-f]{40}$/, '');
```

Comment:
Should we maybe add `#partial-timeline` to the link instead, to automatically scroll down to the latest events (which most likely triggered the notification in the first place)? See also `open-issue-to-latest-comment`.

```suggestion
	link.pathname = link.pathname.replace(/\/files\/[\da-f]{40}..[\da-f]{40}$/, '#partial-timeline');
```

---

I also thought about scrolling to the "New changes since you last viewed" timeline event, but sadly there is no ID we can link to :slightly_frowning_face: 

![grafik](https://github.com/refined-github/refined-github/assets/202916/01e21945-7586-4345-adbb-be77ac29389e)

<details>
<summary>HTML</summary>

```html
<div
  class="js-timeline-item js-timeline-progressive-focus-container rgh-seen-6970248962 rgh-conversation-activity-filtered">
  <div class="TimelineItem">
    <span class="TimelineItem-badge color-bg-accent-emphasis color-fg-on-emphasis">
      <svg aria-hidden="true" height="16" viewBox="0 0 16 16" version="1.1" width="16" data-view-component="true"
        class="octicon octicon-plus">
        <path
          d="M7.75 2a.75.75 0 0 1 .75.75V7h4.25a.75.75 0 0 1 0 1.5H8.5v4.25a.75.75 0 0 1-1.5 0V8.5H2.75a.75.75 0 0 1 0-1.5H7V2.75A.75.75 0 0 1 7.75 2Z">
        </path>
      </svg>
    </span>
    <div class="TimelineItem-body d-flex flex-items-center">
      <strong class="flex-auto">New changes since you last viewed</strong>
      <a href="/stylelint/stylelint/pull/7252/files/94eac4ec4b8914055b9e9d732fd8a92b17a3afdf..HEAD"
        data-ga-click="Pull Requests, view changes button, location:revision marker" data-view-component="true"
        class="btn-sm btn"> View changes
      </a>
    </div>
  </div>
</div>
```

</details>

Reply:
?rgh-changes#partial-timeline

We can catch this URL and scroll to that element via JS